### DISCUSSION TITLE
Unconfirmed Balance

### DISCUSSION THREAD

**Discussion by Prodigygr** [lightningnetwork/lnd#7804] [Troubleshooting]
Please i need some help to understand something. 
Im new LND node operator, with Rasp Pi4 & Umbrel. 10 day ago i start to operate [the node](https://mempool.space/lightning/node/034de3ecea5e03c009ba98add61a4cde6b1b539378be244854d372d3b107afa76c) and opening channels using swaps at [lightningnetwork.plus](https://lightningnetwork.plus/).
At some point, trying to open channel [using this swap](https://lightningnetwork.plus/swaps/17088#comment_94099) with my last sats, and i find out that i have unconfirmed balance. 
Searching at mempool, i have find out, that i have [one transaction ](https://mempool.space/tx/38dfb46a0247f5d0e42f51db42f6aec7fc34d63cd4a0c73e922a67f7106c84c3) with the unconfirmed balance and i do not know who did this transaction.
Also [this transaction](https://mempool.space/tx/38dfb46a0247f5d0e42f51db42f6aec7fc34d63cd4a0c73e922a67f7106c84c3), replaced [with this transaction](https://mempool.space/tx/668b33eb4adb4f596371dd0551e9f5513fc03de847b822b6d1e937aee80d2f7c), after applied RBF, but with 147 sats only. 
I have unconfirmed balance 3.086.250 sats. At the first transaction i see the 3mil sats but in the replacement i dont.
At my ZEUS wallet i can bump the unconfirmed transaction, but the replacement one, with 147sats. 
So were is my 3mil sats? Whho did this transactions?
Sorry for the long script. Im trying to find help and i do not find understanding. 
Thank you all!!!

**Comment by nixonrichard1971** [comment]
I have the same issue. Also some of my channels have force-closed by themselves. Suddenly I have a very large unconfirmed balance in my node and if I look in ride the lightning I have 30+ transactions with the label "0:sweep" and 136 sats; 147 sats etc. If I run the command "lncli listunspent" it shows me all the utxo's with 0 confirmations, but in ride the lightning the 0:sweep transactions that are unconfirmed are of 136 sats; 147 sats... etc. To summarize, it seems the same issue, and to me the unconfirmed balance sums up to at least 50 times the size of the channel that was closed... I'm a bit anxious!

PS: I also did CPFP with ride the lightning when opening some prior channels because I had set the fee too low

**Reply by Prodigygr** [reply]
Thank you for your replay. Im new to this, and no one believe me that this is not my mistake. I have i friend that say to me that "blockchain do not lie". Well, me neither. 
I have also one channel with force close. I know it that is this the whole mess. 
[This channel! ](https://mempool.space/lightning/channel/875616975624994816) Has open chnnel with me with 1mil sats, and then close it. If i remember well, hi has 4-5 more channels open with other nodes, that also close it. 
Im not experience user at node, and i believe some experienced eyes has to look at this.

**Comment by Prodigygr** [comment]
Please help.
Now the transaction not found on the mempool
I have "Transaction not found.
Waiting for it to appear in the mempool..." from the pool.
I have two transactions, one with 147sats, and one with 3,086,250sats [to this address ](https://mempool.space/address/bc1p4k22g0vs09v66jm4sw0f5dvn95mwcvvhxdfcawrsy00lh95jsa7sxmvqa9)
Few hours ago i can see the transaction on the mempool. Now what?
Please help!

**Reply by nixonrichard1971** [reply]
i fixed my problem by rescanning my lnd wallet tx's, see if that helps

**Reply by guggero** [reply]
This should be fixed by https://github.com/lightningnetwork/lnd/pull/7800. But for now, rescanning the wallet (starting `lnd` with `--reset-wallet-transactions`) should fix the problem as well, as @nixonrichard1971 mentioned.

**Reply by Rothschildiuk** [reply]
try to install lightning Shell on your Umbrel, then execute command `lncli wallet publishtx <hex tx of Force Close Chanel>`
you can get hex [here](https://mempool.space/tx/052a6586cfabb55765e2239090e55901b73fc4aa6f9874baff72b7e3716a6b38)

<img width="1023" alt="image" src="https://github.com/lightningnetwork/lnd/assets/15833131/460c80e0-0a8d-4237-aede-1e3ff333c741">
 

https://mempool.space/api/tx/052a6586cfabb55765e2239090e55901b73fc4aa6f9874baff72b7e3716a6b38/hex

**Reply by MykelSIlver** [reply]
This solution solved my issue also. Thank you so much!

**Reply by servietos** [reply]
Hello, can you please tell me what should be the output of command '_lncli wallet publishtx <HEX>_'?
I get only '_"txid": "transaction hash"_'

**Comment by britvr** [comment]
![image](https://github.com/lightningnetwork/lnd/assets/167540638/49527fe8-adcd-4e8c-9c4e-f616b6abda48)
Hey, guy, I'm running into this problem. Does the lnd --reset-wallet-transactions still work?
